client/packages/common/src/intl/context/IntlContext.tsx

```diff
@@ -35,10 +35,11 @@ export function initialiseI18n({
 
   // Served with frontend bundle
   // Electron `main` window translations should be served with relative path
+  // for electron, the preloaded script path is `file://:` we don't get a valid API_HOST url until we connect to the server and re-initialise the window
   const defaultTranslationsLoadPath = `${!!isElectron ? '.' : ''}/locales/{{lng}}/{{ns}}.json`;
 
-  // Served from backend
-  const customTranslationsLoadPath = `${Environment.API_HOST}/custom-translations`;
+  // Served from backend, on electron we use a dummy but valid url https://localhost:8000 which shouldn't actually be used.
+  const customTranslationsLoadPath = `${!!isElectron ? 'https://localhost:8000' : Environment.API_HOST}}/custom-translations`;
```

Comment:
This means custom translations wouldn't be available on Desktop if you run your server on any other port?

Reply:
I don't think it does? Wait let me test...

Reply:
Ok further followup I did get a chance to yarn build then restart but it seems the yarn build version isn't loading custom translations in either a browser or electron. Will need to investigate further tomorrow.